### PR TITLE
Refactor: Update README.md

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,8 @@ event := models.EventModel{
 	Response:     rsp,
 	SessionToken: &sessionToken,
 	Tags:         nil,
-	UserId:       &userId,
+  UserId:       &userId,
```
Comment (dkm199) [inline README.md:95]: are these aligned with lines above ?

File: README.md [hunk 2]
```diff
@@ -164,7 +165,8 @@ event := models.EventModel{
...
```
Comment (dkm199) [inline README.md:168]: same here

**Review by dkm199 — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (keyur9) [reply README.md:95]: Updated the PR which fixes the indentation issue as Markdown document and preview shows different results in VSCode.

**Pull request merged**